src/routes/Tool/Tool.tsx

```diff
@@ -316,27 +316,36 @@ const Tool = () => {
             </>
           )}
           <div className="flex flex-col gap-5 mb-8">
-            {toolStore.tyomahdollisuudet.map((tyomahdollisuus) => {
-              const { id } = tyomahdollisuus;
+            {toolStore.mixedMahdollisuudet.map((mahdollisuus) => {
+              const { id } = mahdollisuus;
               const isFavorite = toolStore.suosikit?.find((s) => s.suosionKohdeId === id) !== undefined;
+              // TODO: mahdollisuus tyyppi will be available in mahdollisuus ehdotukset
+              const mahdollisuusTyyppi =
+                toolStore.tyomahdollisuudet.findIndex((m) => m.id === id) > -1
+                  ? 'TYOMAHDOLLISUUS'
+                  : 'KOULUTUSMAHDOLLOSUUS';
               return (
                 <NavLink
-                  key={tyomahdollisuus.id}
-                  to={`/${i18n.language}/${t('slugs.job-opportunity.index')}/${id}/${t('slugs.job-opportunity.overview')}`}
+                  key={id}
+                  to={
+                    mahdollisuusTyyppi == 'TYOMAHDOLLISUUS'
+                      ? `/${i18n.language}/${t('slugs.job-opportunity.index')}/${id}/${t('slugs.job-opportunity.overview')}`
+                      : `/${i18n.language}/${t('slugs.education-opportunity.index')}/${id}/${t('slugs.education-opportunity.overview')}`
+                  }
                 >
                   <OpportunityCard
                     isFavorite={isFavorite}
                     isLoggedIn={isLoggedIn}
                     toggleSelection={() => toggleMahdollisuus(id)}
                     toggleFavorite={() => void toolStore.toggleSuosikki(id)}
                     selected={toolStore.mahdollisuudet.includes(id)}
-                    name={getLocalizedText(tyomahdollisuus.otsikko)}
-                    description={getLocalizedText(tyomahdollisuus.tiivistelma)}
-                    matchValue={toolStore.tyomahdollisuusEhdotukset?.[id].pisteet}
+                    name={getLocalizedText(mahdollisuus.otsikko)}
+                    description={getLocalizedText(mahdollisuus.tiivistelma)}
+                    matchValue={toolStore.mahdollisuusEhdotukset?.[id]?.pisteet}
                     matchLabel={t('fit')}
-                    type="work"
-                    trend={toolStore.tyomahdollisuusEhdotukset?.[id].trendi}
-                    employmentOutlook={toolStore.tyomahdollisuusEhdotukset?.[id].tyollisyysNakyma ?? 0}
+                    type={mahdollisuusTyyppi == 'TYOMAHDOLLISUUS' ? 'work' : 'education'}
```

Comment:
Same thing as above, use triple equals signs. While it doesn't really affect anything here, it's just better practice.